### PR TITLE
Set ThreadPoolExector max threads to 1

### PR DESCRIPTION
**Pull request by jccraft-uw**
We're still getting HTTP connect timeouts after we switched from 15 to 8 threads. This change sets the max threads to 1. A backoffice change will decrease the frequency with which this make gets invoked.

### CODE REVIEW

**Comment by kairstenfay** [comment]
(For posterity: that backoffice change is https://github.com/seattleflu/backoffice/pull/63)

**Comment by kairstenfay** [comment]
For more context, here's Thomas's original PR attempting to tackle this error: https://github.com/seattleflu/backoffice/pull/63

File: bin/export-record-barcodes [hunk 1]
```diff
@@ -60,7 +60,7 @@ def main():
         ScanProject(23089, "en", "irb-kiosk"),
     ]
 
-    with ThreadPoolExecutor(8) as pool:
+    with ThreadPoolExecutor(1) as pool:
```
Comment (kairstenfay) [inline bin/export-record-barcodes:63]: if we're setting this to 1 thread, do we need `ThreadPoolExecutor()` at all?

Comment (tsibley) [reply bin/export-record-barcodes:63]: Nope, but I would probably leave it. We want to be making these requests in parallel, and I assume we'll be able to do so again in due course.

Comment (kairstenfay) [reply bin/export-record-barcodes:63]: That works for me.

**Pull request merged**